dev/tests/integration/testsuite/Magento/Bundle/_files/fixed_bundle_product_zero_price_option_selection.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\Bundle\Model\Product\Price;
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\Product\Attribute\Source\Status;
+use Magento\Catalog\Model\Product\Type;
+use Magento\Catalog\Model\Product\Visibility;
+use Magento\TestFramework\Bundle\Model\PrepareBundleLinks;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\TestFramework\Workaround\Override\Fixture\Resolver;
+
+Resolver::getInstance()->requireDataFixture('Magento/Bundle/_files/multiple_products.php');
+
+$objectManager = Bootstrap::getObjectManager();
+$productRepository = $objectManager->create(ProductRepositoryInterface::class);
```

Comment:
please use ->get() for ProductRepositoryInterface

Reply:
Done